modules/pubxBidAdapter.js

```diff
@@ -0,0 +1,49 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+const BIDDER_CODE = 'pubx';
+const ALIAS_BIDDER_CODE = ['pbx'];
```

Comment:
Is there a particular reason you are defining an ALIAS? This is a new adapter,yes? An alias is not required and may just complicate things.

Reply:
Thank you @GLStephen, we removed alias as you suggested. Could you please check again?